core/domain/exp_domain.py

```diff
@@ -2224,6 +2228,15 @@ def update_init_state_name(self, init_state_name):
                 'exploration.' % (init_state_name, self.states.keys()))
         self.init_state_name = init_state_name
 
+    def update_auto_tts_enabled(self, auto_tts_enabled):
+        """Update whether automatic text-to-speech is enabled.
+
+        Args:
+            auto_tts_enabled: bool. Boolean representing whether automatic
```

Comment:
Nit: ok to just do `bool. Whether automatic ...`. The "bool" already conveys "boolean".

Reply:
done